lang/po/es_ES.po

```diff
@@ -156222,7 +156222,7 @@ msgid ""
 msgstr ""
 "Un practicante de aikido intermedio puede protegerse contra múltiples oponentes.\n"
 "\n"
-"Daño bloqueado reducido en un 100%% de Destreza.\n"
```

Comment:
This %% was also necessary.

However, now that I'm looking at it, why does "Intermediate Akido" translate to this long phrase?

Reply:
Probably a translation error. `lang/po/es_ES.po` in `master` branch reads
```po
#: lang/json/martial_art_from_json.py
msgid "Intermediate Aikido"
msgstr "Aikido intermedio"
```

Reply:
So I'd better revert the `*.po` changes and only add the checking script in this pull request.